packages/protocol/contracts/devnet/DevnetTaikoL1.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../L1/TaikoL1.sol";
+
+/// @title DevnetTaikoL1
+/// @dev Labeled in AddressResolver as "taiko"
+/// @custom:security-contact security@taiko.xyz
+contract DevnetTaikoL1 is TaikoL1 {
+    /// @inheritdoc ITaikoL1
+    function getConfig() public pure override returns (TaikoData.Config memory) {
+        return TaikoData.Config({
+            chainId: 167_001,
+            blockMaxProposals: 324_000,
+            blockRingBufferSize: 360_000,
+            maxBlocksToVerify: 16,
+            blockMaxGasLimit: 240_000_000,
+            livenessBond: 125e18, // 125 Taiko token
+            stateRootSyncInternal: 16,
+            maxAnchorHeightOffset: 64,
+            basefeeAdjustmentQuotient: 8,
+            basefeeSharingPctg: 75,
+            gasIssuancePerSecond: 5_000_000,
+            ontakeForkHeight: 0
```

Comment:
Let‘s try 20?

Reply:
Lets do `0` by default for client testing purpose? or `1`?

Reply:
Changed to `2`

Reply:
In my opinion, we need to make sure that the logic before and after ontake is correct every time we reset devnet